packages/test-data/tests-unit/math-css-vars/math-css-vars.less

```
@@ -0,0 +1,13 @@
+// Math functions cannot resolve a runtime CSS var() at compile time, so the
+// whole call is left for the browser instead of erroring (issue #4224).
+.trig {
+  a: sin(var(--angle));
+  b: cos(var(--a));
+  c: calc(tan(var(--t)) * 1px);
+  d: atan2(var(--x), var(--y));
+}
+// Concrete numeric arguments still evaluate.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)